docs/admin/receipt_template_editor.adoc

```diff
@@ -228,15 +232,14 @@ once you import the receipts on the new machine.
 
 Exporting templates
 ^^^^^^^^^^^^^^^^^^^
-As you can only save a template on to the computer you are working on you will
+As you can only save a template on to the computer you are working (*Save Locally*) on you will
```

Comment:
Looks like "(*Save Locally*)" split the phrase "working on" in half.  We'll need to move "on" back next to "working".

Reply:
Corrected this.